airbyte-integrations/connectors/source-github/source_github/source.py

```diff
@@ -102,18 +102,40 @@ def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) ->
     def streams(self, config: Mapping[str, Any]) -> List[Stream]:
         authenticator = self._get_authenticator(config["access_token"])
         repositories = self._generate_repositories(config=config, authenticator=authenticator)
-        organizations = list({org.split("/")[0] for org in repositories})
+        organizations = list(set({org.split("/")[0] for org in repositories}))
         full_refresh_args = {"authenticator": authenticator, "repositories": repositories}
         incremental_args = {**full_refresh_args, "start_date": config["start_date"]}
         organization_args = {"authenticator": authenticator, "organizations": organizations}
 
+        # Get the default branch for each repository
+        default_branches = {}
+        repository_stream = Repositories(authenticator=authenticator, organizations=organizations)
```

Comment:
The user repository may not belong to any organization, and in this case we will probably get an error.

Reply:
I think, as variant, in this case, we can expand the `RepositoryStats `stream with the default_branch field and use it, as variant.

Reply:
Ah yes, the Repositories stream actually fails if a non-org repo is specified right now.